### ISSUE TITLE
Elements in tab order that should not be on homescreen, as well as one in editor

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2335]
**Describe the bug**

a few elements are on the home page in tab order when you return to the home page that shouldn't be

This seems to include the toolbar (start, restart, etc) in the homescreen, as well as ``body.main.ms-Fabric--isFocusVisible``.

**To Reproduce**
1. Go to editor
2. return to homescreen
3. try to tab, and see that there are a few elements that are not visible are included in the tab order

**Expected behavior**

Should not include elements that are not visible / interactive in tab order

**Screenshots**

Here are  the elements showing up while in the homescreen:

![2019-08-02 15 10 31](https://user-images.githubusercontent.com/5615930/62401792-950e0c80-b539-11e9-97cf-d0c0250d452a.gif)

of these ones, only the inverted menu should actually be accessible by tabbing
